### PR TITLE
update submitpop

### PR DESCRIPTION
**Pull request by Mr-Leshiy**

### CODE REVIEW

File: src/vbk/rpc_register.cpp [hunk 1]
```diff
@@ -183,42 +183,43 @@ UniValue getpopdata(const JSONRPCRequest& request)
...
-    if (request.fHelp || request.params.size() > 2)
+    if (request.fHelp || request.params.size() > 3)
         throw std::runtime_error(
             "submitpop [vtbs] (atv)\n"
             "\nCreates and submits a PoP transaction constructed from the provided ATV and VTBs.\n"
             "\nArguments:\n"
-            "1. atv       (string, required) Hex-encoded ATV record.\n"
+            "1. atvs      (array, required) Array of hex-encoded ATV record.\n"
             "2. vtbs      (array, required) Array of hex-encoded VTB records.\n"
+            "3. vbk_blocks      (array, required) Array of hex-encoded VbkBlock records.\n"
```
Comment (Warchant) [inline src/vbk/rpc_register.cpp]: make vbk blocks first, atvs last

File: src/vbk/rpc_register.cpp [hunk 2]
```diff
@@ -183,42 +183,43 @@ UniValue getpopdata(const JSONRPCRequest& request)
...
+
+    return payloads;
+}
+
 UniValue submitpop(const JSONRPCRequest& request)
 {
-    if (request.fHelp || request.params.size() > 2)
+    if (request.fHelp || request.params.size() > 3)
         throw std::runtime_error(
             "submitpop [vtbs] (atv)\n"
```
Comment (Warchant) [inline src/vbk/rpc_register.cpp]: also here

File: src/vbk/rpc_register.cpp (hunk 1 above)
Comment (Mr-Leshiy) [reply src/vbk/rpc_register.cpp]: Fixed

File: src/vbk/rpc_register.cpp (hunk 2 above)
Comment (Mr-Leshiy) [reply src/vbk/rpc_register.cpp]: fixed

**Review by Warchant — APPROVED** [review]

**Pull request merged**